app/assets/javascripts/views/mobile_menu.js

```diff
@@ -0,0 +1,43 @@
+(function(App) {
+
+  'use strict';
+
+  var MobileMenu = Backbone.View.extend({
+
+    el: 'body',
+
+    events: {
+      'click .btn-mobile-menu' : 'onClickToggleMenu',
+      'click .btn-close' : 'onClickToggleMenu'
+    },
+
+    initialize: function() {
+      this.status =  new (Backbone.Model.extend({
+        defaults: {
+          hidden: true
+        }
+      }));
+
+      this._listeners();
+    },
+
+    _listeners: function() {
+      this.status.on('change:hidden', this.changedStatus, this);
+    },
+
+    onClickToggleMenu: function() {
+      var hidden = this.status.get('hidden');
+      this.status.set('hidden', !hidden);
+    },
+
+    changedStatus: function() {
+      this.$('html').toggleClass('-menu-open');
+      this.$('body').toggleClass('-menu-open');
+      this.$('.c-mobile-menu').toggleClass('-is-open');
+      this.$('.c-menu').toggleClass('-is-open');
```

Comment:
It is nicer and more solid if we toggle the class in relation with the attribute from the status.